CAM/InvalidateVehicleIdleCam.md

```diff
@@ -5,7 +5,7 @@ aliases: ["_0x9E4CFFF989258472"]
 ## INVALIDATE_VEHICLE_IDLE_CAM
 
 ```c
-// 0x9E4CFFF989258472 0x65DDE8AF
+// _0x9E4CFFF989258472 0x65DDE8AF
 void INVALIDATE_VEHICLE_IDLE_CAM();
```

Comment:
Sorry for me being not enough clear :(
I mean native's name should be prefixed, not hash.

Reply:
Ok I reverted both _ and added _ to the raw native name instead. So if the raw native name doesn't match the hash then we add a underscore to it?

Reply:
Correct. You can review the process in our [contribution guidelines](https://github.com/citizenfx/natives/blob/master/.github/CONTRIBUTING.md) as well. (Which you should have read 😛)

Reply:
Okok well I pushed a new commit fixing it